index.js

```diff
@@ -37,4 +47,4 @@ const stringify = (object, type) => {
     }
 }
 
-module.exports = { parse, stringify }
+module.exports = { parse, stringify, removeFileExt }
```

Comment:
Please don't export the `removeFileExt` function. That's an implementation detail that should be of no concern for people using this module.